controllers/cloud.redhat.com/version.txt

```diff
@@ -1 +1 @@
-main
+v0.4.0-5-gffb46f4
```

Comment:
We don't commit these changes, pre-push should have ignored this, if it doens't please update it to function like the Clowder one!